docs/reference/upgrade.asciidoc

```diff
@@ -47,12 +47,15 @@ a|
 
 [WARNING]
 ====
-The following upgrade paths are *not* supported:
+The following upgrade path is *not* supported (both full cluster restart and rolling upgrade):
 
 * 6.8 to 7.0.
```

Comment:
I don't think a one-item list makes sense here. Can you fold it into a single sentence?